integration-cli/docker_cli_save_load_test.go

```diff
@@ -167,6 +167,20 @@ func (s *DockerSuite) TestSaveAndLoadRepoFlags(c *check.C) {
 	c.Assert(before, checker.Equals, after, check.Commentf("inspect is not the same after a save / load"))
 }
 
+func (s *DockerSuite) TestSaveWithNoExistImage(c *check.C) {
+	testRequires(c, DaemonIsLinux)
+	name := "test-save-image"
+	dockerCmd(c, "run", "--name", name, "busybox", "true")
```

Comment:
I just don't think it's needed; because if that image has never been there, we don't need to create it first, only to delete it later.

perhaps @vdemeester can have a look as well, but I don't think it's needed :smile: